File: LightOJ/1107 - How Cow.java
//1107 - How Cow

import java.util.*;

public class Main{
	public static void main(String[]args){
		int t, m, x, y, x1, y1, x2, y2;
		Scanner sc=new Scanner(System.in);

		t=sc.nextInt();

		for(int i=1; i<=t; i++){
			x1=sc.nextInt();
			y1=sc.nextInt();
			x2=sc.nextInt();
			y2=sc.nextInt();

			m=sc.nextInt();

			System.out.println("Case "+i+":");

			for(int j=1; j<=m; j++){
				x=sc.nextInt();
				y=sc.nextInt();

				if(((x>=x1)&&(x<=x2))&&((y>=y1)&&(y<=y2))){
					System.out.println("Yes");
				}
				else{
					System.out.println("No");
				}
			}
		}
	}
}

File: LightOJ/1107 - How Cow.cpp
//1107 - How Cow

#include <bits/stdc++.h>

using namespace std;

int main(){
    int x1,y1,x2,y2,x,y,m,i,j,t;
   
    cin>>t; 
   
    if(t<=50){
       for(i=1;i<=t;i++){
            cin>>x1>>y1>>x2>>y2;
            cin>>m;
            cout<<"Case "<<i<<":\n";
           
            for(j=1;j<=m;j++){
                cin>>x>>y;
               
                if((x>x1)&&(x<x2)&&(y>y1)&&(y<y2)){
                    cout<<"Yes\n";
                }
                else{
                    cout<<"No\n";
                }
            }  
       }
    }
    return 0;
}